src/backend/distributed/citus.sql

```diff
@@ -103,6 +103,27 @@ SET search_path = 'pg_catalog';
 
 /* master_* functions */
 
+CREATE FUNCTION citus_table_size(relation_name text)
```

Comment:
We define new UDFs in files like citus--6.1-17--6.1-18.sql. Each time you create new UDF or make metadata changes is Citus metadata tables, you will need to create new .sql file and increase citus version. We can talk about this tomorrow in more detail.